src/server/api/schema.js

```diff
@@ -334,8 +353,47 @@ async function editCampaign(id, campaign, loaders, user) {
   return newCampaign || loaders.campaign.load(id)
 }
 
+async function updateInteractionSteps(campaignId, interactionSteps, idMap = {}){
```

Comment:
Does this fix a bug (e.g. maybe this one: https://github.com/MoveOnOrg/Spoke/issues/165) or why is this different from what we had?